README.md

```diff
@@ -84,9 +84,10 @@ This is a *demo* that enables basic, read-only querying of the [Microsoft Graph
 
 ## Limitations/to-dos
 * [x] Translate OData inheritance relationships
+* [x] Enable passing arguments (id for indexing into collections)
```

Comment:
Getting collection items by id is a great start, but is there a more generic approach?

Is there a way to let users query on other properties besides "id"?  Having the GraphiQL editor tell you which fields you can get a collection item on would be awesome!  For example, users can be accessed by "userPrincipalName" or id and worksheets can be selected on "name" or id. Is there an annotation that specifies what fields are indexed?

Reply:
Every entity in Graph derives from the base 'entity' which has a key called 'id'. Alternative methods for fetching items in collections are either function based (which I agree is another feature this demo should support at some point) or are not described in metadata.